app-route.html

```diff
@@ -180,7 +180,9 @@
        */
       __routeQueryParamsChanged: function(queryParams) {
         if (queryParams && this.tail) {
-          this.set('tail.__queryParams', queryParams);
+          if (this.tail.__queryParams != queryParams) {
```

Comment:
oh right i didn't see there are other things done in the `if` :P
btw please use `!==` instead of `!=` in this file